Code/Editor/EditorProcessor/EditorProcessor.cpp

```diff
@@ -179,8 +181,26 @@ class ezEditorApplication : public ezApplication
           }
         }
 
-        QApplication::quit();
-      });
+        QApplication::quit(); });
+
+      const ezInt32 iReturnCode = ezQtEditorApp::GetSingleton()->RunEditor();
+      if (iReturnCode != 0)
+        SetReturnCode(iReturnCode);
+    }
+    else if (opt_Resave.GetOptionValue(ezCommandLineOption::LogMode::AlwaysIfSpecified))
+    {
+      ezQtEditorApp::GetSingleton()->OpenProject(sProject).IgnoreResult();

```

Comment:
I don't think you should ignore this.

Reply:
Copied it from the transform option: OpenProject has an immediate mode which is set to false by default, so it will only queue a project open internally and always returns EZ_SUCCESS.